### ISSUE TITLE
Will RES be updated to support Chrome Manifest V3?

### ISSUE THREAD

**Issue by Pikamander2** [honestbleeps/Reddit-Enhancement-Suite#5350]
It looks like Chrome [is planning to remove](https://www.theregister.com/2021/09/27/google_chrome_manifest_v2_extensions/) support for Manifest V2 extensions in a year or two.

A quick glance at [the manifest files](https://github.com/honestbleeps/Reddit-Enhancement-Suite/search?q=manifest) suggests that RES is still using Manifest V2, and I couldn't find any Issues talking about it.

Sorry if it's already on the radar; I just want to make sure that it doesn't go unnoticed and become a last-minute problem.

**Comment by danielsoaresp** [comment]
Hardly:
https://www.reddit.com/r/RESAnnouncements/comments/sh83gx/announcement_life_of_reddit_enhancement_suite/

**Comment by creesch** [comment]
> Hardly: https://www.reddit.com/r/RESAnnouncements/comments/sh83gx/announcement_life_of_reddit_enhancement_suite/

It is unlike that no attempt will be done to update RES to support manifest V3. As the post states it is in maintenance mode, not quite dead.

**Comment by haykam821** [comment]
Web-accessible resources allow Chrome extensions such as Reddit Enhancement Suite to be [fingerprinted](https://z0ccc.github.io/extension-fingerprints/). It also looks like Chrome will [disallow Manifest v2 extensions starting in 2023](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/).

**Comment by ToxicSmurf** [comment]
> Detecting extensions using web accessible resources is ***not possible on Firefox*** as Firefox extension ID's are unique for every browser instance. Therefore the URL of the extension resources cannot be known by third parties.

If only it didn't use up so many system resources 😭 I want to switch so bad.

**Comment by BenMcGarry** [comment]
Yes https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/5403
